tap_twilio/sync.py

```diff
@@ -327,7 +323,7 @@ def sync_endpoint(
                             # their endpoints will be in the results,
                             # this will grab the child path for the child stream we're syncing,
                             # if we're syncing it. If it doesn't exist we just skip it below.
-                            child_path = record.get('_subresource_uris', {})\
+                            child_path = record.get('links', {})\
```

Comment:
seems like it will break to get the child paths of the old streams where the `_subresource_uris` exists and not the `links`.

Can we make it backward compatible to aware of both option? Something like:
`child_path = record.get('_subresource_uris', record.get('links', {}))`